app/components/LoginForm/ForgotPasswordForm.tsx

```diff
@@ -1,86 +1,56 @@
-import { Button } from '@webkom/lego-bricks';
-import { Component } from 'react';
-import { connect } from 'react-redux';
-import { compose } from 'redux';
-import { reduxForm, Field, SubmissionError } from 'redux-form';
+import { useEffect, useState } from 'react';
+import { Field } from 'react-final-form';
 import { sendForgotPasswordEmail } from 'app/actions/UserActions';
+import { useAppDispatch } from 'app/store/hooks';
 import { createValidator, required, isEmail } from 'app/utils/validation';
-import { Form, TextInput } from '../Form';
-import type { FormProps } from 'redux-form';
+import { Form, LegoFinalForm, SubmitButton, TextInput } from '../Form';
 
-type Props = {
-  sendForgotPasswordEmail: (arg0: { email: string }) => any;
-} & FormProps;
-type State = {
-  submitted: boolean;
-};
+const validate = createValidator({
+  email: [required(), isEmail()],
+});
 
-class ForgotPasswordForm extends Component<Props, State> {
-  mounted = false;
-  state = {
-    submitted: false,
-  };
+const ForgotPasswordForm = () => {
+  const [mounted, setMounted] = useState(false);
+  const [submitted, setSubmitted] = useState(false);
```

Comment:
Might be possible to get these values from `react-final-form` through the formState https://final-form.org/docs/final-form/types/FormState

Reply:
Yes probably - I'll look into it tomorrow

Reply:
Looked into it now. It is possible to look at the submitted state, but it's not possible to get and use these outside of the form.